### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Success Criterion 3.3.8 Redundant entry

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1453]
“Success Criterion 3.3.8 Redundant entry (Level A): For steps in a process, information previously entered by or provided to the user that is required on subsequent steps is either:

- auto-populated, or
- available for the user to select.
- Exception: When re-entering the information is essential.”

Most of our comments/proposals are requests to: consider updating the SC title to Populate prior data, explicitly clarify that auto-populating security info is different from verification, and consider cross linking to the WCAG 2.2 Accessible Authentication SC3.3.7, consider our suggestions about additional examples of approved exceptions and sufficient techniques, add a section for failure examples, and further expand the reference made to a Turing test.

Please find detailed specifics covered in our [3.3.8 Redundant entry (Level A) Google Doc](https://docs.google.com/document/d/1N9J4FaJFVD00bHxI_aVkrEHEShBcHsH0YTxqxN7Yoeo/edit).

**Comment by alastc** [comment]
Hi @dshoukry,

Tackling the comments in order of the document, and with associate PR #1639:

DRAFT response: 

<hr>

> Can this be renamed to Populate prior data or something else that is more clear and easier to remember?

I'll put that to the group.

> Under Success Criterion 3.3.7 Accessible Authentication, though, auto-populating some security info is recommended. Could it be made clearer that auto-populating security info is different from verification, maybe by cross linking to the earlier success criterion?

The scenario is different, where Accessible Authentication would by nature apply across sessions, and entering a password would be 'essential' to the purpose of that form. There is an extra paragraph saying that in the PR.

> Do repeated entries in search fields also apply here?

In the context of a process, going to a search page would be considered the 'start' of a process, and would not need to pre-fill the previous entries. However, if the second page of a search process (i..e. the results page) would need to pre-fill the results if the input is associated with the previous search. That seems like a reasonable example to add.

**Comment by dshoukry** [comment]
Thank you for all of the clarifications, accepting all of our content changes, and the updated PRs. 

BTW, I've just returned from some time off so I'm catching up on all of the latest updates. In an attempt to be helpful and acknowledge where we are with all of the changes, I'll be summarizing any outstanding questions in this and each of the five other original SC-level issues that are still open. 

For this Issue: if it'd be easier for you, we can close this out and you don't need to follow up here with the group's decision? We recognize this is a tricky one to name given the exception for information entered between sessions and appreciate the decision either way. We'll be able to figure out the final name outcome when the SC gets published. :-)

**Comment by alastc** [comment]
We discussed a couple of the individual comments today:

New name: We appreciate the name isn't ideal, but there wasn't a better option that got consensus to make a change.

PR #1640 updated the definition of Turing test to link through to the W3C page on CAPTCHA.

PR #1639 has been merged in which deals with some of the other comments.

I suggest reviewing the new version when it comes out.
